index.js

```diff
@@ -588,6 +584,13 @@ Router.prototype._defaultRoute = function (req, res, ctx) {
   }
 }
 
+Router.prototype.setDefaultRoute = function (defaultRoute) {
+  if (defaultRoute) {
+    assert(typeof defaultRoute === 'function', 'The default route must be a function')
+    this.defaultRoute = defaultRoute
+  }
+}
```

Comment:
I would implement it with a `defineProperty` with a getter/setter instead. Note that this might not be needed at all as somebody could already do `router.defaultRoute = fn` - so maybe this PR is not needed?

Reply:
True, it's possible to update it directly inside `fastify` and expose the methods needed by `fastify-websocket` there.
I thought it would be more correct to do it here to keep its validation (assert) encapsulated here without repeating it inside `fastify`, but yeah this doesn't seem necessary after all.

Reply:
I would just update `router.defaultRoute` in fastify.

Reply:
Got it, thanks